.circleci/config.yml

```diff
@@ -181,7 +181,7 @@ jobs:
             pip install --user codespell
       - run:
           name: Check spelling
-          command: ~/.local/bin/codespell -S "*.enc,.git" -I ./scripts/codespell_whitelist.txt
+          command: ~/.local/bin/codespell -S "*.enc,.git,Dockerfile*" -I ./scripts/codespell_whitelist.txt
```

Comment:
Any other or better suggestions? It complained about "keyserver": https://circleci.com/gh/ethereum/solidity/158189

Reply:
I think this is the only way to do it AIUI, as codespell doesn't treat Dockerfile syntax differently. (for example we might still want to spell-check comments within this file).

Reply:
That's what I thought, yep.